examples/python-sdk/context/file_search_server/main.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+"""
+File Search Server Sample
+
+A simple HTTP server that provides AI-powered file search using FileSystem Context.
+Search results are processed with AI to summarize only the relevant results.
+
+Usage:
+    python examples/context/file_search_server/main.py [workspace-directory]
+
+Endpoints:
+    GET  /search?q=<query>  - Search for files and get AI-summarized results
+    GET  /health - Health check
+"""
+
+import json
+import sys
+from datetime import datetime
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from pathlib import Path
+from urllib.parse import parse_qs, urlparse
+
+# Add parent directory to path for imports
+sys.path.insert(0, str(Path(__file__).parent))
```

Comment:
I wonder if we should just inline search_handler.py to avoid this weird sys.path insert?

Or add `__init__.py`, if that works?